# initial position does not have to be correct if you are not flying with flowdeck or crazyswarm

#c1 = 193
#d1 = 209
#e1 = 225
#f1 = 241

crazyflies:
- channel: 125
  id: 2
  initialPosition:
  - 0.5
  - 0.5
  - 0.0
  type: default
  
- channel: 125
  id: 4
  initialPosition:
  - -0.5
  - -0.5
  - 0.0
  type: default

- channel: 105
  id: 201
  initialPosition:
  - 0.5
  - -0.5
  - 0.0
  type: default

- channel: 95
  id: 209
  initialPosition:
  - -0.5
  - 0.5
  - 0.0
  type: default

- channel: 95
  id: 210
  initialPosition:
  - 0.0
  - 0.0
  - 0.0
  type: default

# - channel: 95
#   id: 211
#   initialPosition:
#   - 0.0
#   - 0.0
#   - 0.0
#   type: default

# - channel: 105
#   id: 199
#   initialPosition:
#   - 0.5
#   - -0.5
#   - 0.0
#   type: default